front/public/locales/en/operationalStudies/project.json

```diff
@@ -0,0 +1,49 @@
+{
+  "affairCode": "Affair code",
+  "budget": "Budget",
+  "createdOn": "Created on",
+  "createStudy": "Create a study",
+  "description": "description",
+  "error": {
+    "unableToPostDocumentTitle": "Picture upload failed",
+    "unableToPostDocument": "Picture loading impossible"
+  },
+  "filterPlaceholder": "Search…",
+  "fundedBy": "Funding",
+  "geremiCode": "GEREMI code",
+  "modifiedOn": "Updated on",
+  "modifyProject": "Update project",
+  "objectives": "objectives",
+  "openStudy": "Open",
+  "projectBudget": "Estimated budget",
+  "projectCancel": "Cancel",
+  "projectCreateButton": "Create the projet",
+  "projectCreationTitle": "Create a project",
+  "projectDeleteButton": "Delete",
+  "projectDeleted": "Successful suppression!",
```

Comment:
To suppress is closer to abolir, I prefer deletion here, but as you like ;)

Reply:
yes, done ;)